pages/tools/cli.md

```diff
@@ -582,65 +582,82 @@ continuously stream output
 
 # Sync
 
-## sync [source]
+## sync [uuid]
 
 WARNING: If you're running Windows, this command only supports `cmd.exe`.
 
 Use this command to sync your local changes to a certain device on the fly.
 
-The `source` argument can be either a device uuid or an application name.
+After every 'resin sync' the updated settings will be saved in
+'<source>/.resin-sync.yml' and will be used in later invocations. You can
```

Comment:
@shaunmulligan what if resin sync detected `resin-sync.yml` and converted it to `.resin-sync.yml` after prompting the user accordingly? The new `.resin-sync.yml` is compatible with the older format although some options are ignored:

- `progress` and `verbose`, being debugging options are no longer persisted 
- `source` is ignored - `.resin-sync.yml` is now expected to always be in the root of the project directory to be synced